seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json

```diff
@@ -813,11 +809,11 @@
         "console.log(addFriend(user, 'Pete'));"
       ],
       "tests": [
```

Comment:
The below tests need to be switched around (sorry, the first review may have been a little unclear), the 3 showing as removed are the latest work, they are properly formatted with code tags and the instructions will not match the tests if the third test is omitted. 

Just for the record, even though EverNote may not be doing so well, I love this tool that you recommended!

![image](https://cloud.githubusercontent.com/assets/18563015/22490876/c40cf2a6-e7ed-11e6-9c04-bd07fcf0e936.png)

Other than that, everything looks good to me, and this should be a simple change. I would fix it myself, but not sure how to add to your PR without write access to your repo.